src/index.js

```diff
@@ -12,5 +15,7 @@ import './assets/fonts/fontello/css/fontello.css';
 
 angular
   .module('app', [adbViewModule, 'ui.router', 'pascalprecht.translate'])
+  .constant('_', _)
```

Comment:
Was lodash in use previously? The only files that use it in this PR are `soum-data.service.js` and `histogram.directive.js`, and both of those bring it in as an Angular dependency. Grepping through the repo it looks like the only other place that uses _ is cartodb's test file.